composer.json

```diff
@@ -14,7 +14,8 @@
     ],
     "require": {
         "php": ">=5.5.0",
-        "atk4/dsql": "*"
+        "atk4/dsql": "*",
+        "atk4/core": "1.0"
```

Comment:
because we're using implementation for hooks, containers and initializers:

http://agile-core.readthedocs.io/en/latest/